### PR TITLE
Add inline error handling and tests for tag creation/editing

### LINKED ISSUE #5684: [Bug] Tag Page tries to save tags with no name
The tag page allows to add tags and edit existing ones. It currently fails to prevent empty tags from being saved.

Steps to reproduce:

1. Add a new Tag
2. Make sure it is saved
3. Clear out the name field
4. Tab out of the field
5. Watch the console log:  "TypeError: Cannot read property 'message' of undefined" (It tries to show a notification but the server responds with the tag)

Optimally this should be caught on the client side with inline validation and prevent the tag from being saved.

### PR DESCRIPTION
**Pull request by kevinansfield**
closes #5684 
- add inline error checking and display to tag name and meta data fields
- add tests for tags screen

### CODE REVIEW

**Comment by ErisDS** [comment]
Tested this and it's looking good with the tiny exception that it needs to be marked as closes #5684 rather than #5484 :wink:

**Comment by kevinansfield** [comment]
Rebased and fixed the issue tag :)

File: core/client/app/templates/settings/tags/settings-menu.hbs [hunk 1]
```diff
@@ -53,17 +54,19 @@
 
                 <div class="settings-menu-content">
                     <form>
-                    <div class="form-group">
+                    {{#gh-form-group errors=activeTag.errors property="meta_titel"}}
```
Comment (ErisDS) [inline core/client/app/templates/settings/tags/settings-menu.hbs]: Argh! I was waiting for this to go green and then I gave it one last look and spotted this... sorry!

**Comment by kevinansfield** [comment]
Ooh, good catch! Fixed and added checks for field error states that should have caught that typo.

**Pull request merged**